plugin-usage/adding-content/item-properties/basic.md

```diff
@@ -4,6 +4,11 @@ description: Collection of basic item properties
 
 # Basic
 
+{% hint style="warning" %}
+<mark style="color:red;">Don't forget the namespace!</mark>\
+Don't forget to define a namespace for every of your config!
```

Comment:
```suggestion
Don't forget to define a namespace for every part of your config!
```

Reply:
Tbh namespace is not neccessary to define in every PART of the config but in every config

Reply:
in my eyes, it's basically saying the same thing, though maybe it'd be better to reformat it a bit. maybe like

```suggestion
Don't forget to define a namespace for all of your configs!
```